### PR TITLE
prov/efa: Replace mr access modes

### PR DESCRIPTION
**Pull request by dipti-kothari**
Signed-off-by: Dipti Kothari <dkothar@amazon.com>

### CODE REVIEW

File: prov/efa/src/rxr/rxr_pkt_type_req.c [hunk 1]
```diff
@@ -1199,7 +1199,7 @@ void rxr_pkt_handle_rtr_recv(struct rxr_ep *ep, struct rxr_pkt_entry *pkt_entry)
 	rx_entry->window = rtr_hdr->read_req_window;
 	rx_entry->iov_count = rtr_hdr->rma_iov_count;
 	err = rxr_rma_verified_copy_iov(ep, rtr_hdr->rma_iov, rtr_hdr->rma_iov_count,
-					FI_SEND, rx_entry->iov);
+					FI_REMOTE_WRITE, rx_entry->iov);
```
Comment (wzamazon) [inline prov/efa/src/rxr/rxr_pkt_type_req.c]: Otherway around, FI_SEND -> FI_REMOTE_READ. FI_RECV-> FI_REMOTE_WRITE

Comment (dipti-kothari) [reply prov/efa/src/rxr/rxr_pkt_type_req.c]: Fixed!

**Comment by wzamazon** [comment]
bot:aws:retest

**Comment by dipti-kothari** [comment]
> It's better to update the commit message so that it does not exceed the character limit per line.

Updated. Thanks

**Pull request merged**